boards/nucleo-l552ze-q/doc.txt

```diff
@@ -5,14 +5,30 @@
 
 ## Flashing the device
 
-The ST Nucleo-L552ZE-Q board includes an on-board ST-LINK V2.1 programmer. The
-easiest way to program the board is to use OpenOCD. Once you have installed
-OpenOCD (look [here](https://github.com/RIOT-OS/RIOT/wiki/OpenOCD) for
-installation instructions), you can flash the board simply by typing
+The ST Nucleo-L552ZE-Q board includes an on-board ST-LINK programmer and can be
+flashed using OpenOCD.
+@note The upstream version of OpenOCD doesn't contain yet support for this board,
+so the source code version from http://openocd.zylin.com/#/c/5510
+must be built to be able to flash this board (adapt the configure command with
+your preferred installation directory):
+
+```
+$ git clone https://github.com/ntfreak/openocd.git
```

Comment:
```suggestion
$ git clone https://git.code.sf.net/p/openocd/code openocd
```

Why not use the original repo or the [official mirror](https://repo.or.cz/w/openocd.git)

Reply:
Makes sense to use the official repo, I changed the PR in that direction.

Reply:
Using https://repo.or.cz/w/openocd.git directly didn't work (`fatal: https://repo.or.cz/w/openocd.git/info/refs not valid: is this a git repository?`), I finally applied your initial suggestion.